metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/RunExecutionAthenaAggregator.java

```diff
@@ -106,7 +107,7 @@ protected String createQuery(AthenaTablePartition partition) {
         final Select<Record> runExecutionsFromTasks = select(taskExecutionFields)
                 .from(dedupedTaskExecutions);
 
-        return DSL.using(SQLDialect.DEFAULT, new Settings().withRenderFormatted(true))
+        return DSL.using(SQLDialect.DEFAULT, new Settings().withRenderFormatted(true).withStatementType(StatementType.STATIC_STATEMENT))
```

Comment:
Because the synthesized SQL statement now contains constants, we have to tell JOOQ to make a static statement, otherwise the constants will be parameterized (appear in the statement with a placeholder which will be substituted at submission time).